### PR TITLE
Added changelogs for 1.8.1 release

### PR DESCRIPTION
**Pull request by swathinsankaran**
This PR contains the changelogs for the 1.8.1 release as well as the controller version max version to 22.1.1.

### CODE REVIEW

**Comment by swathinsankaran** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

File: CHANGELOG.md [hunk 1]
```diff
@@ -257,4 +257,22 @@ All notable changes to this project will be documented in this file. The format
...
+### Added
+ - AKO now claims support for Kubernetes 1.24.
+ - Support for `AviInfraSetting` CRD in Passthrough Ingresses/Routes.
+ - Support for ISTIO mTLS authentication. (Tech-preview)
+ - Support for IPv4/IPv6 dual-stack networking. (Tech-preview)
+
+### Changed
+ - Added a knob to block the processing of objects in system namespaces in OpenShift and K8s clusters.
+ - For Insecure Ingress deployments, priority labels associated with pools will be case insensitive.
+ - In `NodePortLocal` deployment, AKO processes only the PODs with the annotation `nodeportlocal.antrea.io`.
```
Comment (akshayhavile) [inline CHANGELOG.md:273]: @swathinsankaran : AKO already do line number 273 and 274. But AKO used to do that in later in graph stage and in 1.8.1 we changed it in ingestion layer.  So we can present  these changes as an optimization or skip it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -257,4 +257,22 @@ All notable changes to this project will be documented in this file. The format
...
+## AKO-1.8.1
+
+### Added
+ - AKO now claims support for Kubernetes 1.24.
+ - Support for `AviInfraSetting` CRD in Passthrough Ingresses/Routes.
+ - Support for ISTIO mTLS authentication. (Tech-preview)
+ - Support for IPv4/IPv6 dual-stack networking. (Tech-preview)
+
+### Changed
+ - Added a knob to block the processing of objects in system namespaces in OpenShift and K8s clusters.
```
Comment (akshayhavile) [inline CHANGELOG.md:271]: We can add this as part of `Added` section.

File: CHANGELOG.md (hunk 1 above)
Comment (swathinsankaran) [reply CHANGELOG.md:273]: It will be better to remove this as it was already supported.

File: CHANGELOG.md (hunk 2 above)
Comment (swathinsankaran) [reply CHANGELOG.md]: I have addressed the comment.

**Review by aaha97 — APPROVED** [review]
Lgtm

**Review by akshayhavile — APPROVED** [review]

**Comment by akshayhavile** [comment]
build ako

**Review by aaha97 — APPROVED** [review]

**Comment by swathinsankaran** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Comment by swathinsankaran** [comment]
build ako

**Pull request merged**